zubhub_frontend/zubhub/src/components/Navbar/Navbar.jsx

```diff
@@ -113,7 +113,7 @@ function PageWrapper(props) {
         } else if (searchType === SearchType.PROJECTS) {
           completions = await api.autocompleteProjects({ query, token });
           completions = completions
-          .filter(c=>( c.creator.id === props.auth.id && c.publish.type !== 1 ))
+          .filter(c=> props.auth.id ? (c.creator.id === props.auth.id && c.publish.type !== 1): c )
```

Comment:
Thanks for that remark @coderatomy . I have adjusted that to `props.auth.token` even though there is actually an `id` property in the auth object, I verified that. Also, @srish marked this as `good-first` and `high-priority` so that's why I selected it to work on it.